docs/enterprise-solutions/configuration/infrastructure-configuration/mcp/overview.mdx

```diff
@@ -0,0 +1,571 @@
+---
+title: "MCP Integration"
+sidebarTitle: "Overview"
+description: "Configure Model Context Protocol (MCP) servers and marketplace integrations for enterprise Cline deployments"
+---
+
+Model Context Protocol (MCP) provides standardized communication between AI models and external data sources, tools, and services. Enterprise MCP integration allows you to securely connect Cline to your organization's systems while maintaining governance and compliance.
```

Comment:
If you're going in the right direction with this paragraph, I would recommend adding a little bit of what I mentioned in the comment above. And please link to the general MCP documentation in the docs files so that they can get access to more context about what MCP servers are about.